pkg/yurtadm/util/kubernetes/util.go

```diff
@@ -144,20 +144,35 @@ func CheckAndInstallKubelet(kubernetesResourceServer, clusterVersion string) err
 		klog.Infof("Cni dir %s already exist, skip install.", constants.KubeCniDir)
 		return nil
 	}
+
+	////download and install kubernetes-cni
+	//cniUrl := fmt.Sprintf(constants.CniUrlFormat, constants.KubeCniVersion, runtime.GOARCH, constants.KubeCniVersion)
+	//savePath := fmt.Sprintf("%s/cni-plugins-linux-%s-%s.tgz", constants.TmpDownloadDir, runtime.GOARCH, constants.KubeCniVersion)
+	//klog.V(1).Infof("Download cni from: %s", cniUrl)
+	//if err := util.DownloadFile(cniUrl, savePath, 3); err != nil {
+	//	return err
+	//}
+	//
+	//if err := os.MkdirAll(constants.KubeCniDir, 0600); err != nil {
+	//	return err
+	//}
+	//if err := util.Untar(savePath, constants.KubeCniDir); err != nil {
+	//	return err
+	//}
+
 	//download and install kubernetes-cni
-	cniUrl := fmt.Sprintf(constants.CniUrlFormat, constants.KubeCniVersion, runtime.GOARCH, constants.KubeCniVersion)
-	savePath := fmt.Sprintf("%s/cni-plugins-linux-%s-%s.tgz", constants.TmpDownloadDir, runtime.GOARCH, constants.KubeCniVersion)
+	cniUrl := "https://github.com/openyurtio/openyurt/releases/download/v0.7.0/openyurt-cni-0.8.7-0.x86_64.rpm"
```

Comment:
If using constants, it is better to define in const.
If configuration is supported, it is better to use fmt.Sprintf.

Reply:
openyurt-cni-0.8.7-0.x86_64.rpm(https://github.com/openyurtio/openyurt/releases/download/v0.7.0/openyurt-cni-0.8.7-0.x86_64.rpm) is only seen on https://github.com/openyurtio/openyurt/releases, and it has not changed. 
It seems that it has little to do with the openyurt version. 
Which one should be constants here? It seems that the openyurt version cannot be obtained in the `yurtadm join` codes. Even if it can be obtained, if it is `latest`  version, there is no link like https://github.com/openyurtio/openyurt/releases/download/`latest`/openyurt-cni-0.8.7-0.x86_64.rpm

Reply:
I replaced it with `cniUrl := fmt.Sprintf("%s", constants.OpenYurtCniUrl)`